patches/net/minecraft/client/sounds/MusicManager.java.patch

```diff
@@ -0,0 +1,43 @@
+--- a/net/minecraft/client/sounds/MusicManager.java
++++ b/net/minecraft/client/sounds/MusicManager.java
+@@ -24,22 +_,28 @@
+     }
+ 
+     public void tick() {
+-        Music music = this.minecraft.getSituationalMusic();
++        Music music = net.neoforged.neoforge.client.ClientHooks.selectMusic(this.minecraft.getSituationalMusic(), this.currentMusic);
```

Comment:
This patch is generally too large. The only thing you really need to do is provide a separate if check that if `music` is null and `currentMusic` is not null to stop the current music and set the next song delay to 0. You can return after this.

Reply:
ok, fixed this one